### ISSUE TITLE
panic: interface conversion

### ISSUE THREAD

**Issue by 4bo** [megaease/easegress#239]
I've created two little object to test aggretaion:

```
# egctl object list
- filters:
  - kind: APIAggregator
    mergeResponse: true
    name: agg-api
    pipelines:
    - name: pipeline-demo
    - name: pipeline-demo1
    - name: pipeline-demo2
  flow:
  - filter: agg-api
    jumpIf: {}
  kind: HTTPPipeline
  name: pipeline-api
- cacheSize: 0
  certBase64: ""
  certs: {}
  http3: false
  https: false
  keepAlive: true
  keepAliveTimeout: 60s
  keyBase64: ""
  keys: {}
  kind: HTTPServer
  maxConnections: 10240
  name: server-http
  port: 8000
  rules:
  - host: ""
    hostRegexp: ""
    paths:
    - backend: pipeline-api
      headers: []
      pathPrefix: /api
      rewriteTarget: ""
  tracing: null
  xForwardedFor: false
```
using ```curl http://localhost:8000/api```, i got:
```
server_1  | panic: interface conversion: interface {} is nil, not *supervisor.ObjectEntity
server_1  | 
server_1  | goroutine 391 [running]:
server_1  | github.com/megaease/easegress/pkg/object/trafficcontroller.(*TrafficController).GetHTTPPipeline(0xc000f33c80, 0x28ac8c5, 0x7, 0xc000197490, 0xe, 0x0, 0x0)
server_1  |     github.com/megaease/easegress/pkg/object/trafficcontroller/trafficcontroller.go:550 +0x1b9
server_1  | github.com/megaease/easegress/pkg/object/rawconfigtrafficcontroller.(*RawConfigTrafficController).GetHTTPPipeline(0xc000f3e400, 0xc000197490, 0xe, 0xc001022f40, 0xc0000528f6, 0x2cf6268)
server_1  |     github.com/megaease/easegress/pkg/object/rawconfigtrafficcontroller/rawconfigtrafficcontroller.go:95 +0x5c
server_1  | github.com/megaease/easegress/pkg/filter/apiaggregator.(*APIAggregator).handle.func1(0xc000fc5610, 0xc001207d40, 0xc00150fad0, 0x3, 0x3, 0x2, 0xc000197490, 0xe, 0xc000ffa600)
server_1  |     github.com/megaease/easegress/pkg/filter/apiaggregator/apiaggregator.go:205 +0x92
server_1  | created by github.com/megaease/easegress/pkg/filter/apiaggregator.(*APIAggregator).handle
server_1  |     github.com/megaease/easegress/pkg/filter/apiaggregator/apiaggregator.go:203 +0x18d

```

**Comment by benja-wu** [comment]
Hi @4bo 
* This `panic` dues to the missing of `pipeline-demo`, `pipeline-demo1`, and `pipeline-demo2` in your Easegress instance. And it had been fixed after `V1.2.0`. Can u help us to update your code locally? Because in the panic message, `trafficcontroller.go:550`, it's not an interface value casting anymore. [Here](https://github.com/megaease/easegress/blob/main/pkg/object/trafficcontroller/trafficcontroller.go#L550) is what the code looks like now.
* The root cause of this missing is that Easegress' TrafficController can't find the target pipeline inside the system. Please follow the cookbook step by step, Thx.

**Comment by 4bo** [comment]
thank you @benja-wu
as the issue is fixed in ```v1.2.1```, i close this
